app/models/pending_upload.rb

```diff
@@ -17,14 +21,25 @@ def path
     copied_file_name
   end
 
+  def container?
+    !type.empty? && type.first == "container"
```

Comment:
This should perhaps refer to a string constant namespaced to the `BrowseEverything` module or a class within it.